app/subscriber/src/features/daily-overview/AVOverviewPreview.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Col, IAVOverviewInstanceModel, IReportResultModel, Loading, Show } from 'tno-core';
+
+import { useAVOverviewInstances } from './hooks/useAVOverviewInstances';
+import * as styled from './styled';
+
+const AVOverviewPreview: React.FC = () => {
+  const [{ findAVOverview, previewAVOverview }] = useAVOverviewInstances();
+
+  const [isLoading, setIsLoading] = React.useState(true);
+  const [, setInstance] = React.useState<IAVOverviewInstanceModel>();
+  const [preview, setPreview] = React.useState<IReportResultModel | undefined>();
+  const [isPublished, setIsPublished] = React.useState(false);
+
+  const handlePreviewReport = React.useCallback(async () => {
+    try {
+      setIsLoading(true);
+      const instance = await findAVOverview(new Date().toISOString());
```

Comment:
Are they not allowed to view prior days too?

Reply:
At the moment I just based it off the UI, provided but I can create another ticket to cover this!